python/kwiver/vital/types/metadata.cxx

```diff
@@ -114,16 +114,21 @@ PYBIND11_MODULE( metadata, m )
   .def( "__nonzero__", &unknown_metadata_item::is_valid )
   .def( "__bool__",    &unknown_metadata_item::is_valid )
   .def_property_readonly( "tag",  &unknown_metadata_item::tag )
+  .def_property_readonly( "type", [] ( unknown_metadata_item const& self )
+  {
+    return "NoneType";
```

Comment:
Running `inst.type is None` will return `False` with this check... Try using the older version: `get_type()`. Pybind11 should have functionality for this somewhere if this doesnt work

Reply:
You want `type()` to return `None`? Shouldn't we just return `py::none()` then?

Reply:
No, we want `type()` to return `NoneType` here. If you run `type(None)` from the Python interpreter, you get `<class 'NoneType'>`. Right now, `data` returns `None`, so we want `type` to return the equivalent of running `type(None)` from the interpreter. Doing something like `return py::none().get_type()` might do the trick.

Reply:
I changed it to `py::none().get_type()`.